lib/vanilla-masker.js

```diff
@@ -89,6 +89,19 @@
 
   VanillaMasker.prototype.maskMoney = function(opts) {
     this.opts = mergeMoneyOptions(opts);
+
+    // Ensures the number are in format \d+.{precision}
+    var value,
+        precision = this.opts.precision;
+    for (var i = 0, len = this.elements.length; i < len; i++) {
```

Comment:
I think this approach is not the best one available. In this scenario you are considering every single element under VanillaMasker should be bootstrapped into the precision. (Because not every element is necessarily a `toMoney` implementation) If that is really the way of approaching it, then this should live under the general bootstrap. Not maskMoney only.